### PR TITLE
Added expandable card option to expander component

### PR DESCRIPTION
**Pull request by barcafa**

### CODE REVIEW

**Comment by barcafa** [comment]
Current look on docs
Captura de Pantalla 2021-05-19 a les 12 48 29

File: projects/demo/src/app/demo/pages/expand-page/expander-page.component.html [hunk 1]
```diff
@@ -52,6 +66,14 @@ <h3>Inputs</h3>
             <dt>contentLoaded</dt>
             <dd>Any value marking expander's content as loaded or updated so the expand can update its body height.</dd>
         </dl>
+        <dl>
+            <dt>card</dt>
+            <dd>By default card expander will be collapsed and can only be expanded by the expander button</dd>
```
Comment (mpellerin42) [inline projects/demo/src/app/demo/pages/expand-page/expander-page.component.html:71]: I feel like you should update the description on the top of the page to present the two options. Saying something like:
```
By default, expander are displayed like in a list: the toggle button being on the left. The <code>card</code> option allows to display a card expander with the toggle button on the right.
```

```suggestion
            <dd>Card option allows to change the expander look and feel by displaying toggle button on the right and any <code>pa-expander-header-side-block</code> on the left. When this option is set, the card expander is collapsed by default and can only be expanded by the expander button</dd>
```

**Review by mpellerin42 — CHANGES_REQUESTED** [review]

File: projects/pastanaga-angular/src/lib/expander/expander.component.ts [hunk 2]
```diff
@@ -68,6 +68,10 @@ export class ExpanderComponent implements AfterViewInit, OnDestroy {
         this.terminator.complete();
     }
 
+    onTitleClick() {
+        if (!this.card) this.toggleExpand();
```
Comment (mpellerin42) [inline projects/pastanaga-angular/src/lib/expander/expander.component.ts:71]: we don't inline if statement.

File: projects/demo/src/app/demo/pages/expand-page/expander-page.component.html [hunk 3]
```diff
@@ -2,6 +2,9 @@
     <pa-demo-title>Expander</pa-demo-title>
     <pa-demo-description>
         <p>Expander component allows to expand/collapse a section by clicking on its header or on the chevron button next to it.</p>
+        <p>By default, expander are displayed like in a list: the toggle button being on the left. The <code>card</code>
```
Comment (mpellerin42) [inline projects/demo/src/app/demo/pages/expand-page/expander-page.component.html:5]: typo: `expander is` or `expanders are` 😅

**Review by mpellerin42 — APPROVED** [review]

**Pull request merged**